### ISSUE TITLE
Win7 64-bit, openbazaar-3.0 beta gives socket.error

### ISSUE THREAD

**Issue by greatwolf** [OpenBazaar/OpenBazaar#1183]
I downloaded the openbazaar-beta-3.0 zipped binary for windows [here](https://github.com/OpenBazaar/OpenBazaar/wiki/Windows-support).

The full error stacktrace when I run it from an admin cmd:

    G:\openbazaar\openbazaar>openbazaar.exe
    Traceback (most recent call last):
      File "openbazaar.py", line 31, in <module>
      File "node\openbazaar.pyo", line 16, in <module>
      File "node\openbazaar_daemon.pyo", line 14, in <module>
      File "twisted\internet\reactor.pyo", line 39, in <module>
      File "twisted\internet\selectreactor.pyo", line 196, in install
      File "twisted\internet\selectreactor.pyo", line 72, in __init__
      File "twisted\internet\base.pyo", line 498, in __init__
      File "twisted\internet\posixbase.pyo", line 289, in installWaker
      File "twisted\internet\posixbase.pyo", line 84, in __init__
      File "socket.pyo", line 224, in meth
    socket.error: [Errno 10051] A socket operation was attempted to an unreachable network

Is this a known problem?

**Comment by hoffmabc** [comment]
I've not seen this, but it certainly doesn't seem like anything that should crash the app.

**Comment by andre-amorim** [comment]
I saw that ...keeping rebooting the system ..just now came to my mind to
set a tap to monitoring if there is some malformed package ...

**Comment by greatwolf** [comment]
Just wanted to give an update on my issue. The problem was coming from a Microsoft loopback driver under my network adapters. For some reason, that was preventing my machine from making localhost connections with itself.

For example, I also had issues with Tor not being able to receive packets on ports its listening to when that packet originates from my localhost machine. Another big giveaway that helped me diagnose the issue was when even NetCPS failed to do a bandwidth benchmark when I tried to connect to myself.

I finally resolved this socket problem by removing the microsoft loopback altogether and all the problems described above went away.
